setup.ts

```diff
@@ -456,7 +456,19 @@ async function main(): Promise<void> {
     message: "Are you setting up this project using Docker?",
     default: false,
   });
-  if (!isDockerInstallation) {
+  
+  if (isDockerInstallation) {
+    const DB_URL = "mongodb://localhost:27017/talawa-api";
+    const config = dotenv.parse(fs.readFileSync(".env"));
```

Comment:
1. Why is this file here?
2. We have an issue to add mongoDB support that can be used instead. It's better to have only one setup file. We don't want this to fail later
    1.  https://github.com/PalisadoesFoundation/talawa-api/issues/1605

Reply:
1. But in docker installation, we can't detect the mongodb instance during setup time as containers go up after npm run setup.
2. I have added this because there was a bug in docker installation where importing sample data throws an erorr because it can't find mongodb url in .env file.
3. In docker installation, the mongodb url is the same every time, isn't it better to hard code it?

Reply:
Removed changes done to setup.ts and informed the person working on the issue.